lib/person.rb

```diff
@@ -1 +1,78 @@
 # your code goes here
+class Person
+  attr_reader :name
+  attr_accessor :bank_account
+  attr_reader :happiness
+  attr_reader :hygiene
+
+  def initialize(name)
+    @name = name
+    @bank_account = 25
+    @happiness = 8
+    @hygiene = 8
+  end
+
+  def happiness=(value)
+      if value > 10
+        @happiness = 10
+      elsif value < 0
+        @happiness = 0
+      else 
+        @happiness = value
+      end
+  end
+
+  def hygiene=(value)
+    if value > 10
+      @hygiene = 10
+    elsif value < 0
+      @hygiene = 0
+    else 
+      @hygiene = value
+    end
+  end
+    
+    def happy?
+      @happiness > 7  
```

Comment:
Don't need the `@` since you have `attr_reader`s
```suggestion
      happiness > 7  
```